routing/routing_integration_tests/routing_integration_tests.pro

```diff
@@ -13,6 +13,11 @@ TEMPLATE = app
 ROOT_DIR = ../..
 DEPENDENCIES = map routing search storage indexer platform geometry coding base osrm jansson protobuf tomcrypt succinct stats_client
 
+# this dependency is not built on Linux
```

Comment:
And it shouldn't. Why opening_hours is not build on Linux should be explained in 3party/3party.pro